### ISSUE TITLE
names of gates in `++  si` of `hoon.hoon`

### ISSUE THREAD

**Issue by joshuareagan** [urbit/arvo#739]
The gates in the `++  si` (signed integer) core are given nonstandard names; e.g. `sum`, `dif`, `pro` in place of the usual `add`, `sub`, `mul` etc.  Why?  `++  rs`, `++  rd`, `++  rq` all have the usual names for mathematical functions, it seems like `++  si` should too.

**Comment by keatondunsford** [comment]
I don't have any background context on this, but this seems like a reasonably good catch.

**Comment by cgyarvin** [comment]
I was probably thinking of a collision, which isn’t an actual concern.  At the same time, we definitely want to batch these kinds of changes into a clear revision process...

**Comment by joshuareagan** [comment]
Process will be important because e.g., `sum` is used all over the place in `hoon.hoon`, and probably in other places too.  Can I assume for the purpose of writing docs for the future that this change will happen?

**Comment by cgyarvin** [comment]
Yeah, you’re right, there really is no harm in doing it incrementally — but it should go in via the research branch.

**Comment by ohAitch** [comment]
I think it is collisions, namely so that within the core it is clear which op is atom vs signed without resorting to `^`

**Comment by joshuareagan** [comment]
Ok, I see that it uses `++  ff` which is made for the IEEE float standard.  We'd have to figure something else out.  Which still shouldn't be that hard...
